Detectors/TPC/workflow/include/TPCWorkflow/RecoWorkflow.h

```diff
@@ -59,32 +61,41 @@ framework::WorkflowSpec getWorkflow(std::vector<int> const& tpcSectors,
                                     bool propagateMC = true, unsigned nLanes = 1, //
                                     std::string const& cfgInput = "digitizer",    //
                                     std::string const& cfgOutput = "tracks",      //
-                                    int caClusterer = 0                           //
+                                    int caClusterer = 0,                           //
+									int zs = 0,
+									int zs10bit = 0,
+									float zsThreshold = 2.0f
 );
 
 static inline framework::WorkflowSpec getWorkflow(std::vector<int> const& tpcSectors,           //
                                                   bool propagateMC = true, unsigned nLanes = 1, //
                                                   std::string const& cfgInput = "digitizer",    //
                                                   std::string const& cfgOutput = "tracks",      //
-                                                  int caClusterer = 0                           //
+                                                  int caClusterer = 0,                           //
+												  int zs = 0,
+												  int zs10bit = 0,
+												  float zsThreshold = 2.0f
 )
 {
   // create a default lane configuration with ids [0, nLanes-1]
   std::vector<int> laneConfiguration(nLanes);
   std::iota(laneConfiguration.begin(), laneConfiguration.end(), 0);
-  return getWorkflow(tpcSectors, laneConfiguration, propagateMC, nLanes, cfgInput, cfgOutput, caClusterer);
+  return getWorkflow(tpcSectors, laneConfiguration, propagateMC, nLanes, cfgInput, cfgOutput, caClusterer, zs, zs10bit, zsThreshold);
 }
 
 static inline framework::WorkflowSpec getWorkflow(bool propagateMC = true, unsigned nLanes = 1, //
                                                   std::string const& cfgInput = "digitizer",    //
                                                   std::string const& cfgOutput = "tracks",      //
-                                                  int caClusterer = 0                           //
+                                                  int caClusterer = 0,                           //
+											      int zs = 0,
```

Comment:
To me, the variable names have become a bit confusing because we have two types of zero suppression:
- Reading zero suppressed raw data as input.
- Running zero-suppression on the fly on the digits.
Perhaps, I would call zs here zsOnTheFly and the other one zsRaw (I think it should already by called so anyway).

Reply:
Indeed, this was quite misleading. I did the renaming as proposed.